expected-consensus.md

```diff
@@ -224,7 +268,7 @@ Tickets = append(Tickets, newTicket)
 
 // If the current ticket isn't a winner and the block isn't found by another miner,
 // derive a ticket from the last ticket
-for !winning(electionProof) && !blockFound()) {
+for !IsProofAWinner(electionProof) && !blockFound()) {
  newTicket = VRF(VDF(H(newTicket)))
  newElectionProof = Sig(H(ticketFromRound(curRound-K)))
```

Comment:
shouldn't this use the newTicket?

Reply:
newTicket is generated from one round back (to prove proper delay) whereas electionProof is from k blocks back (with k the lookback parameter for randomness). That's why we use electionProof here. Does that make sense?